pkg/minikube/registry/drvs/virtualbox/virtualbox.go

```diff
@@ -52,6 +52,11 @@ func init() {
 }
 
 func configure(cc config.ClusterConfig, n config.Node) (interface{}, error) {
+	hostOnlyCIDR = strings.Trim(hostOnlyCIDR, "'\"")
+	ip, network, err := net.ParseCIDR(hostOnlyCIDR)
+	if err != nil {
+		return errors.Wrap(err, "invalid address")
```

Comment:
This should be

```
return nil, errors.Wrap(err, "invalid address")
```

for this code to compile

Reply:
Thanks. I'll change it